_posts/static/2017-12-08-getting-started.md

```diff
@@ -53,21 +55,29 @@ Once you assign the devices to the Fleet, all the policies will be immediately a
   * Minimun of numbers
   * Wipe the device at a number of failed attempts to unlock the device
 * Encryption of Internal Storage
-* Peripherals
+* Disable
   * Cameras
   * Wifi
   * Bluetooth
   * GPS
+  * USB
+  * Mobile line
+  * MMS and SMS
+  * Speakerphone
+  * Roaming
+  * FM Radio
+  * NFC
+* And many more.
```

Comment:
we need to check each policy listed here. Some policies were removed for technical reasons recently.

https://github.com/flyve-mdm/glpi-plugin/pull/310/files

- FM radio removed

Reply:
I'll check the list and make the chnages